specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/stable/2022-03-01/examples/GetExtension.json

```diff
@@ -39,6 +39,7 @@
             "omsagent.env.clusterName": "clusterName1"
           },
           "provisioningState": "Creating",
+          "installedVersion": "0.1.4",
```

Comment:
Similar to the above comment, set this to null, since provisioningState is still Creating